assets/themes/base16.tmTheme

```diff
@@ -3,8 +3,8 @@
 <plist version="1.0">
     <dict>
         <!--
-        The colors in this theme are encoded as #RRGGBBAA where RR is an ANSI
-        palette number from 00 to 0f, and AA is the special value 00 to indicate
+        The colors in this theme are encoded as #RRGGBBAA where RR is an base16
```

Comment:
@sharkdp I think this should remain "ANSI palette number." The [base16 numbers](https://github.com/chriskempson/base16/blob/master/styling.md) are mapped onto the [ANSI palette](https://en.wikipedia.org/wiki/ANSI_escape_code#3/4_bit) as follows by base16 terminal templates (e.g. [iTerm2](https://github.com/martinlindhe/base16-iterm2/blob/master/templates/default.mustache)):

| ANSI | Base16 |
|---|---|
| 0 | 0 |
| 1 | 8 |
| 2 | B |
| 3 | A |
| 4 | D |
| 5 | E |
| 6 | C |
| 7 | 5 |
| 8 | 3 |
| 9 | 9 |
| A | 1 |
| B | 2 |
| C | 4 |
| D | 6 |
| E | F |
| F | 7 |

The RR values in this file are from the first column, not the second.